### PR TITLE
refactor: update `blas/ext/base/sapxsumkbn` to follow current project conventions

### LINKED ISSUE #1511: [RFC]: refactor `blas/ext/base/sapxsumkbn` to follow current project conventions
### Description

This RFC proposes refactoring [`@stdlib/blas/ext/base/sapxsumkbn`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/blas/ext/base/sapxsumkbn) to follow current project conventions.

These conventions are outlined in more detail in https://github.com/stdlib-js/stdlib/issues/788. Namely, we want to migrate from C++ add-on interfaces to C add-on interfaces and then make various style and simplification changes.

Some example commits which perform most, if not all, of the requested changes include https://github.com/stdlib-js/stdlib/commit/d66d80cfcbab1da95cb5ae8cfae4a1fc4c5129d1 (saxpy),  https://github.com/stdlib-js/stdlib/commit/3c82fe7b6c0f9fd6ef02c86d6e2024d708ac8c14 (cswap), and https://github.com/stdlib-js/stdlib/commit/5b15c9675be434c0e11b1ec1e1f78d7b089abbcd (dasum).

### Related Issues

Related issues #788, #1152.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by AuenKr**
Resolves #1511.

## Description

> What is the purpose of this pull request?

This pull request:

-   refactored `@stdlib/blas/ext/base/sapxsumkbn` to follow current project conventions

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #1511 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/blas/ext/base/scusumkbn/README.md [hunk 1]
```diff
@@ -65,18 +65,15 @@ The function has the following parameters:
 -   **y**: output [`Float32Array`][@stdlib/array/float32].
 -   **strideY**: index increment for `y`.
 
-The `N` and `stride` parameters determine which elements in `x` and `y` are accessed at runtime. For example, to compute the cumulative sum of every other element in `x`,
+The `N` and `stride` parameters determine which elements in the strided arrays are accessed at runtime. For example, to compute the cumulative sum of every other element in the strided input array,
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/blas/ext/base/scusumkbn/README.md]:
```suggestion
The `N` and stride parameters determine which elements in the strided arrays are accessed at runtime. For example, to compute the cumulative sum of every other element in the strided input array,
```

File: lib/node_modules/@stdlib/blas/ext/base/scusumkbn/docs/repl.txt [hunk 2]
```diff
@@ -3,8 +3,8 @@
     Computes the cumulative sum of single-precision floating-point strided array
     elements using an improved Kahan–Babuška algorithm.
 
-    The `N` and `stride` parameters determine which elements in `x` and `y` are
-    accessed at runtime.
+    The `N` and `stride` parameters determine which elements in the strided
```
Comment (Planeshifter) [inline lib/node_modules/@stdlib/blas/ext/base/scusumkbn/docs/repl.txt]:
```suggestion
    The `N` and stride parameters determine which elements in the strided
```

File: lib/node_modules/@stdlib/blas/ext/base/sapxsumkbn/README.md [hunk 3]
```diff
@@ -56,16 +55,14 @@ The function has the following parameters:
 -   **x**: input [`Float32Array`][@stdlib/array/float32].
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/ext/base/sapxsumkbn/README.md:56]:
```suggestion
-   **alpha**: scalar constant.
-   **x**: input [`Float32Array`][@stdlib/array/float32].
```

If you see mistakes like this, feel free to correct.

File: lib/node_modules/@stdlib/blas/ext/base/sapxsumkbn/src/addon.c [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+static napi_value addon( napi_env env, napi_callback_info info ) {
+	STDLIB_NAPI_ARGV( env, info, argv, argc, 4 );
+	STDLIB_NAPI_ARGV_INT64( env, N, argv, 0 );
+	STDLIB_NAPI_ARGV_FLOAT( env, alpha, argv, 1 );
+	STDLIB_NAPI_ARGV_INT64( env, stride, argv, 3 );
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT32ARRAY( env, X, N, stride, argv, 2 );
+
+	napi_value v;
+	napi_status status;
+ 	status = napi_create_double( env, (double)stdlib_strided_sapxsumkbn( N, alpha, X, stride ), &v );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/blas/ext/base/sapxsumkbn/src/addon.c]:
```suggestion
	status = napi_create_double( env, (double)stdlib_strided_sapxsumkbn( N, alpha, X, stride ), &v );
```

**Review by kgryte — APPROVED** [review]
LGTM. Thanks, @AuenKr!

**Pull request merged**